soccer/ui/themes/nyan.stylesheet

```diff
@@ -0,0 +1,36 @@
+/*
+	Copyright 2013 Emanuel Claesson
```

Comment:
We should probably not have this header :P

Reply:
for some reason its in all of the stylesheets :/

Reply:
ah, it was part of the dark stylesheet since the existing sheets were derived from that, since this is original, I'll remove it.